config/redirects.js

```diff
@@ -1539,7 +1539,7 @@ module.exports = [
   },
   {
     from: '/tutorials/bulk-importing-users-into-auth0',
-    to: '/users/migrations/bulk-import',
```

Comment:
So probably all of these most recent URLs that are changed in this file should also redirect, so like

```
{
    from: ['tutorials/bulk-importing-users-into-auth0', '/users/migrations/bulk-import'],
    to: 'users/guides/bulk-user-import'
}
```

To be honest the old `bulk-importing-users-into-auth0` probably could be ditched and just redirect from `bulk-import` to the new one, but there's no real harm in catching both and sending to the new page.